### DISCUSSION TITLE
How to update the bot on sparked host

### DISCUSSION THREAD

**Discussion by Dragoncat91** [jagrosh/MusicBot#1653] [Questions]
I was here yesterday asking how to stop from getting update DMs, changed it to false and I still got another DM today. Whatever. I guess I'm updating it now. Last time I got sparked host staff to update it for me, but I'd like to know how to update it myself. What files do I replace, is it safe to delete old versions of said files etc.

**Comment by gothikieros** [comment]
If you feel like it you can download the source code from the release, edit the file: 
src/main/java/com/jagrosh/jmusicbot/Listener.java

And comment out the following lines:
```java
if(latestVersion!=null && !currentVersion.equalsIgnoreCase(latestVersion))
{
String msg = String.format(OtherUtil.NEW_VERSION_AVAILABLE, currentVersion, latestVersion);
owner.openPrivateChannel().queue(pc -> pc.sendMessage(msg).queue());
}
```

Then use maven to recompile the source code and take the full jar when it is built
